README.md

```diff
@@ -737,8 +749,22 @@ _NOTE: Skip these steps if you are deploying without promoting changes._
 
    The build should start immediately. You can monitor its progress from the
    [Gitlab Pipelines page](https://gitlab.dev.explore.data.humancellatlas.org/azul/azul/pipelines).
+   
+   If reindexing and promoting to staging or production, send a second 
+   warning about reindexing to the #data-wrangling channel at this point.
 
-3. Assuming the build is successful, run
+
+3. If you haven't yet, activate your virtual environment and run
```

Comment:
```suggestion
3. Activate your virtual environment and run
```

Doesn't hurt to run twice but makes instructions simpler.

Reply:
Good call.